SpotifySearch/SpotifyTableViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  SpotifyTableViewController.swift
+//  SpotifySearch
+//
+//  Created by Margaret Ikeda on 10/28/16.
+//  Copyright © 2016 Margaret Ikeda. All rights reserved.
+//
+
+import UIKit
+internal let SpotifyTableViewCellIdentifier: String = "SpotifyCell"
```

Comment:
All these variables should have lower camel-casing.

`"Spotify.json"` is not a .json file in the project.